### PR TITLE
footer-updated

### PR DESCRIPTION
**Pull request by ramyashreeshetty**
review #4

### CODE REVIEW

**Comment by Kanika637** [comment]
Hey @ramyashreeshetty, Kindly give some screenshots of the work you did on the footer🚀

**Comment by ramyashreeshetty** [comment]
image

File: src/Footer.js [hunk 1]
```diff
@@ -1,22 +1,158 @@
...
+            <li className="leading-5">
+              <a href="#">Help</a>
+            </li>
+          </ul>
+        </div>
+      </div>
+      <div className='footer-line'>
+      </div>
+        <div className="rest-footer-2">
+            <img src="" alt="logo"></img>
```
Comment (Jai-Solania-29) [inline src/Footer.js:128]: Hey @ramyashreeshetty, I can see here that in the src section you have not assigned any image, Is that so?

Comment (ramyashreeshetty) [reply src/Footer.js:128]: I had assigned it. It wasn't working so cleared it out.

Comment (ramyashreeshetty) [reply src/Footer.js:128]: I tried again it isnt working, i'll drop the image over here.

amazon-logo

**Comment by Kanika637** [comment]
@ramyashreeshetty, Will work on that image later, I can merge your PR right now but you have some conflicts, kindly check👀

**Comment by ramyashreeshetty** [comment]
Hey resolved it.

**Review by Kanika637 — COMMENTED** [review]
LGTM🚀

**Comment by Kanika637** [comment]
Merging @ramyashreeshetty 🚀
Do give a ⭐ to the repo if you learned something👀

**Pull request merged**